### PR TITLE
Docker Deployment

### PR DESCRIPTION
**Pull request by bcongdon**
Docker deployment for getting new devs setup with a working environment.

### CODE REVIEW

File: database_setup.sql [hunk 1]
```diff
@@ -0,0 +1,5 @@
+CREATE DATABASE IF NOT EXISTS groot_credits_service;
+CREATE DATABASE IF NOT EXISTS groot_recruiter_service_dev;
+CREATE DATABASE IF NOT EXISTS groot_meme_service;
+CREATE DATABASE IF NOT EXISTS groot_user_service_dev;
+CREATE DATABASE IF NOT EXISTS groot_caffeine_service_dev;
```
Comment (sameetandpotatoes) [inline database_setup.sql]: Shouldn't quotes be here too? (`groot_quote_service_dev` if I remember correctly)

**Comment by sameetandpotatoes** [comment]
This should be merged too: https://github.com/acm-uiuc/groot-manifest/pull/4

**Comment by narendasan** [comment]
Would this run all the services in a single container?

**Comment by bcongdon** [comment]
Yup. We can expand on this later to have each service have it's own container, but I wanted it to be simple for getting new devs onboard

**Comment by narendasan** [comment]
Also are you matching versions of languages and packages that we are currently using (e.g go 1.7). might prevent compatibility errors

**Pull request merged**